app/controllers/verify/review_controller.rb

```diff
@@ -23,6 +23,14 @@ def confirm_current_password
     def new
       idv_session.params.symbolize_keys!
       analytics.track_event(Analytics::IDV_REVIEW_VISIT)
+
+      phone_of_record_msg = ActionController::Base.helpers.content_tag(
+        :strong, t('idv.messages.phone.phone_of_record')
+      )
+
+      # TODO: why doesn't this correctly output html without `html_safe`?
```

Comment:
so should we not use `sanitize` here? I'm don't think I understand how this will work by just passing a string, at a minimum we need to pass a the translation key and the `phone_message` that this particular translation requires.

So in the helper we would have to do something like:
```
# controller
flash[:success] = ['foo.bar.baz', { phone_message: phone_message }]

#view
- if msg.present? && msg.is_a(Array)
   div class="alert alert-#{name}" role='alert'
        = t(msg.first, *msg.rest) # I think this is valid?
```

That feels sort of cumbersome but maybe its necessary to avoid calling `sanitize`?